### ISSUE TITLE
[Bug] `hln down` throws error

### ISSUE THREAD

**Issue by yuyicai** [h8r-dev/heighliner#171]
Version:
```
❯ hln version
hln 0.3.0 (579542f) darwin/arm64
```

Errors: 
```
❯ hln down gin-next-2022-05-10t15-58-12
16:04:44	INFO	cmd/down.go:116	patch finalizer to app gin-next-2022-05-10t15-58-12-backend
16:04:44	INFO	cmd/down.go:116	patch finalizer to app gin-next-2022-05-10t15-58-12-frontend
16:04:44	INFO	cmd/down.go:116	patch finalizer to app loki
16:04:44	INFO	cmd/down.go:116	patch finalizer to app nocalhost
16:04:44	INFO	cmd/down.go:116	patch finalizer to app prometheus
16:04:44	INFO	cmd/down.go:133	delete gin-next-2022-05-10t15-58-12-backend...
16:04:44	ERROR	cmd/root.go:93	exit status 1
Error loading state: the secret name tfstate-default- is invalid, a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
This is a requirement for Kubernetes secret names.
The workspace name and key must adhere to Kubernetes naming conventions.
```

**Comment by vgjm** [comment]
This is fixed in #170 . We should make a new release.

**Comment by vgjm** [comment]
I have released v0.3.1. Wait some minitus and upgrade hln.
@yuyicai
